sections/8-units.tex

```diff
@@ -20,8 +20,9 @@ \subsection{Units and Time Scales}
 The \package{sunpy.sun.constants} subpackage contains many standard constants relevant to solar physics with additional information such as uncertainty and reference.
 
 Time is another fundamental quantity that must be appropriately specified for scientific uses.
-Similar motivations as for units led to the adoption of SEP-0008 \citep{sep-0008} which mandates the use of a modern scientific time format throughout the \sunpypkg code base.
-This requirement goes beyond what is needed by the general \python community and is not satisfied by the standard \package{datetime} subpackage.
-\sunpypkg uses the \package{astropy.time} subpackage which provides support for many different time scales, including Coordinated Universal Time (UTC), Terrestrial Time (TT), or International Atomic Time (TAI).
-In the same manner as pairing numbers with their units, a time class pairs a time representation with its time scale, and allows for conversions to other time scales.
-Thus, functions that require precise times (e.g., for ephemeris calculations) can ensure that there is no confusion about the input.
+Solar data analysis requires certain functionality to manipulate and represent times and dates, such as having support for leap seconds and the ability to represent specific time scales and formats. 
+This requirement goes beyond that provided by the standard \python \package{datetime} subpackage.
+For this reason, SEP-0008 \citep{sep-0008} was mandated to adopt the use of the \package{astropy.time} subpackage to represent a date or time throughout the \sunpypkg code base.
+The \package{astropy.time} subpackage provides the necessary support non-UTC time scales used in some solar data sets such as International Atomic Time (TAI), and also provides functionality for representing time formats used in solar physics such as Julian day and Unix time. 
+It also provides support for creating custom time scales and formats. 
+Furthermore the adoption of the \package{astropy.time} within \sunpypkg provides the necessary support for leap seconds, light travel calculations, and high precision time representation. 
```

Comment:
Do you want to add something about being able to accurately compare the timing of solar events to astrophysical events? I'm not sure what the use case would be. Maybe being able to accurately track solar events through the solar system compared to planetary bodies or accurately compare inputs to the Earth's magnetosphere by solar and extra solar events? Another one might be to be able to differentiate events in x-ray data that are solar compared to those that are extra-solar like pulsar bursts.